dv/uvm/cover.cfg

```diff
@@ -1 +1,4 @@
 +tree core_ibex_tb_top.dut
+begin tgl
+  -tree core_ibex_tb_top.dut
```

Comment:
Do you need a .* at the end?

Reply:
Do I need that even if I'm just referring to the top level dut?

Reply:
I think you need this as you only want to disable the toggle coverage below the top level.

Reply:
Yes, you're right, it's been updated.